src/pages/Groups/Group3/styles/Screen2.styled.ts

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components'
+
+export const MainContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 5%;
```

Comment:
![image](https://user-images.githubusercontent.com/63994179/137004483-3e3f9514-d143-4210-82d2-a1fe565a5294.png)
Same thing, margin not needed in MainContainer (should not have space above the top nav bar), place the margin in the components within the container if it is needed instead (usual practice when styling rhapp screens)

Reply:
fixed in commit 12f31c0